client/src/app/project-form/project-form.component.ts

```diff
@@ -95,7 +79,67 @@ export class ProjectFormComponent implements OnInit {
     this.title = this.createMode ? 'New Project' : 'Edit Project';
     this.subtitle = this.createMode ? 'Please provide initial information about your HCA project.\n' +
       '  You will be able to edit this information as your project develops.' : '';
-    this.config.showCancelButton = !this.createMode;
+
+    this.userAccount$ = this.ingestService.getUserAccount();
+    this.userAccount$
+      .subscribe((account) => {
+        this.userIsWrangler = account.isWrangler();
+        this.setUpProjectForm(this.userIsWrangler, layout, this.createMode, pathVariables);
+      });
+  }
+
+  setUpProjectForm(userIsWrangler: boolean, tabLayout: MetadataFormLayout, createMode: boolean, pathVariables: ParamMap) {
+    let tabs = tabLayout.tabs;
+    if (!userIsWrangler) {
+      tabs = tabs.filter(tab => tab.key !== 'project_admin');
+    }
+
+    tabLayout.tabs = tabs;
+    this.setTabLayout(tabLayout);
+    this.createFormConfig(this.getTabLayout(), createMode);
```

Comment:
We should fix the duplication in this ticket: https://github.com/ebi-ait/dcp-ingest-central/issues/225

Reply:
yes, and I've created another ticket for fixing the duplication in the layout, as this duplication involves the project summary view as well: https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/293

Reply:
I tried to refactor `project-form.component.ts` a little so that it would be easier to remove the duplication later on